### PR TITLE
Adds response from James Bourbeau 

### PR DESCRIPTION
**Pull request by jrbourbeau**
Here are my responses. @mrocklin feel free to let me know if I should expand on any of these points.

### CODE REVIEW

File: icecube-cosmic-rays.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+valuable insight into our understanding of the origin of cosmic rays.
+
+
+### How does Dask help?
+
+My analysis pipeline consists of several steps: data preprocessing,
+training/validating machine learning models, performing statistical analyses,
+implementing custom algorithms, etc. Dask provides a user-friendly framework for
+both parallelizing each of these steps as well as seamlessly scaling to large
+amounts of data when required.
```
Comment (mrocklin) [inline icecube-cosmic-rays.md]: If possible I'd like to extract more details from you here :)  Can I ask you to expand a bit on the parts of Dask that you use?  

For example, do you use Dask dataframe to load data from CSV files and then do data preprocessing?

Do you use dask.delayed to build custom algorithms?  If so, what kinds of algorithms?  Specificity here is good.

When you scale to large amounts of data what kind of data and how much?

Comment (jrbourbeau) [reply icecube-cosmic-rays.md]: Yeah, I can definitely add some more details here 

> Do you use dask.delayed to build custom algorithms? If so, what kinds of algorithms? 

Mostly, I used dask.delayed when dealing with embarrassingly parallel problems. I can add a particular example of this. 

> When you scale to large amounts of data what kind of data and how much?

That actually varies quite a bit. Sometimes I'm only analyzing a few GB (e.g. a 3GB dataframe) and other times I'll need to work with a 100GB dataset.

**Comment by jrbourbeau** [comment]
I've added some more details around how I use dask. Let me know if you have any comments, I'd be happy to add to / re-word again

**Comment by mrocklin** [comment]
Thanks @jrbourbeau !

**Pull request merged**